pkg/workload/workload.go

```diff
@@ -165,9 +166,12 @@ func DatumSize(x interface{}) int64 {
 	}
 	switch t := x.(type) {
 	case int:
-		return int64(math.Log10(float64(t)))
+		if t < 0 {
+			t = -t
+		}
+		return int64(bits.Len(uint(t))+8) / 8
```

Comment:
Don't bother eating CI for this, but I'd be less confused if this was ` int64(bits.Len(uint(t))+7) / 8` or had a comment about how this didn't need to be exact. Or if the function were called `ApproxDatumSize`.

Reply:
It's +8 so int64(0) results in 1. This is used to batch things, so I don't want a table of all 0s get infinite size batches. I like `ApproxDatumSize`. I'm already eating CI in https://github.com/cockroachdb/cockroach/pull/22558, so I'll add a comment and do the rename there. TFTR!